playground/src/pages/CodingEnvironment.tsx

```diff
@@ -38,13 +42,24 @@ function CodingEnvironment(): JSX.Element {
    */
   const [proof, setProof] = useState<Uint8Array>(new Uint8Array(0));
 
+  /**
+   * This helps us to show the Debug Menu.
+   */
+  const [showDebug, setShowDebug] = useState(false);
+
+  /**
+   * This sets the debugExecutor such that we can store it for a session.
+   */
+  const [debugExecutor, setDebugExecutor] = useState<DebugExecutor | null >(null);
+
   /**
    * This handles a change in the selected example.
    * If a new example is selected using the dropdown, the inputs and 
    * the code are updated.
    */
   const [, setExample] = React.useState<string>();
   const handleSelectChange = async (exampleChange: string) => {
+    setShowDebug(false);
```

Comment:
I think it makes sense to remove the executor state too. perhaps moves this to an utility function, say, `disable_debug`, that would also call `setDebugExecutor(null)`?